dali/operators/segmentation/random_object_bbox.cc

```diff
@@ -162,6 +162,38 @@ void RandomObjectBBox::FindLabels(std::unordered_set<int> &labels, const T *data
   }
 }
 
+
+template <typename BlobLabel, typename T>
+void RandomObjectBBox::FindLabels(SampleContext<BlobLabel> &ctx,
+                                  const TensorView<StorageCPU, const T> &in) {
+  ctx.labels.clear();
+  int64_t N = in.num_elements();
+  if (!N)
+    return;
+
+  constexpr int64_t chunk_size = 1<<16;
```

Comment:
Does it make sense to put this constant as a class variable with appropriate comment, so it's more visible if any tuning was needed?

Reply:
I think it's overdoing thing - as you can see, actual chunk size may be much larger. Perhaps I'll rename it to `min_chunk_size` or sth.